sql_generators/active_users/templates/focus_android_view.sql

```diff
@@ -2,8 +2,8 @@ CREATE OR REPLACE VIEW
   `{{ project_id }}.{{ app_name }}.active_users_aggregates`
 AS
 SELECT
-  * REPLACE (COALESCE(country, '??') AS country),
```

Comment:
Should this normalization be removed here?

Reply:
Yep, this PR removes it because is already part of the [query.](https://github.com/mozilla/bigquery-etl/blob/176982845d8948ee7ef1cdc5998c6b5fb8f33ac7/sql_generators/active_users/templates/focus_android_query.sql#L127)